externals.json

```diff
@@ -1,6 +1,6 @@
 {
     "npm": {
-        "vsts-task-lib": "0.7.4"
+        "vsts-task-lib": "2.2.0"

```

Comment:
@omeshp - do you see any concerns with bumping up vsts-task-lib so close to on-prem release?